x-pack/plugins/ingest_manager/server/saved_objects/index.ts

```diff
@@ -119,8 +119,7 @@ const savedObjectTypes: { [key: string]: SavedObjectsType } = {
     },
     mappings: {
       properties: {
-        id: { type: 'keyword' },
-        name: { type: 'text' },
+        name: { type: 'keyword' },
```

Comment:
`id` was removed as it's not used (saved object ID is separate and does not need to be declared on mappings)
`name` changed to `keyword` to enable sorting on this field